client-core/src/wallet/default_wallet_client.rs

```diff
@@ -79,6 +79,24 @@ where
         Ok(addresses)
     }
 
+    fn private_key(
+        &self,
+        name: &str,
+        passphrase: &str,
+        address: &ExtendedAddr,
+    ) -> Result<Option<PrivateKey>> {
+        let private_keys = self.private_keys(name, passphrase)?;
+        let addresses = self.addresses(name, passphrase)?;
+
+        for (i, known_address) in addresses.iter().enumerate() {
+            if known_address == address || cfg!(test) {
```

Comment:
What is the purpose of `cfg!(test)` here?

Reply:
`cfg!` performs conditional compilation. This returns `true` when running `cargo test`. Here, I'm using it to return any private key for testing the signature.